circle.yml

```diff
@@ -17,3 +18,14 @@ deployment:
     owner: verygoodsecurity
     commands:
       - mvn deploy -DskipTests=true
+  release:
+      tag: /.*/
+      commands:
+        - git config user.name "circleci"
+        - git fetch
+        - git checkout $RELEASE_BRANCH
+        - git pull origin $RELEASE_BRANCH
+        - git reset --hard
+        - git tag -d $CIRCLE_TAG
```

Comment:
Without this release fails. gitflow will create tag with same name. Its only on circleci machine without pushing